client/js/store/statement/Statement.js

```diff
@@ -211,9 +211,9 @@ function setStatementAssignee (statement) {
   return statement
 }
 export default {
```

Comment:
since `assessmentTable` has been changed to `AssessmentTable`, in `getStatementAction` in `Statement.js`, we need to change the line
`view_mode: rootState.assessmentTable.viewMode,`

to

`view_mode: rootState.AssessmentTable.viewMode,`

(The assessment table is not loading)